dwave/system/temperatures.py

```diff
@@ -249,373 +391,736 @@ def maximum_pseudolikelihood_temperature(bqm=None,
+            for single parameter esimation ``len(bqms)=1``; for multi-parameter
+            estimation only value False is supported.
+        use_jacobian: By default (True) the second derivative of the
+            pseudolikelihood is calculated and used by ScipY root finding
+            methods. The associated complexity of this non-essential
+            calculation is quadratic in len(bqms); use of the second derivative
+            is disabled by setting the value to False.
+    Returns:
+        Tuple: The optimal parameters and a list of bootstrapped estimates (x_estimate, x_bootstrap_estimates):
+
+        * *x_estimate*: parameter estimates
+        * *x_bootstrap_estimates*: a numpy array of bootstrap estimators
+
+    See also:
+        The function :class:`~dwave.system.temperatures.maximum_pseudolikelihood_temperature`
+        https://doi.org/10.3389/fict.2016.00023
+
+        https://www.jstor.org/stable/25464568
+
+    """
+
+
+    root_results = None
+
+    if degenerate_fields is None:
+        if en1 is None:
+            degenerate_fields = len(bqms) == 1
+        else:
+            degenerate_fields = en1.ndim == 2
+
+    en1, sample_weights = _get_en1_for_pseudo_likelihood(en1, bqms, sampleset, sample_weights)
+
+    max_excitation = np.max(en1, axis=(-1, -2))
+    min_excitation = np.min(en1, axis=(-1, -2))
+    prod_minmax = max_excitation * min_excitation
+
+    if (en1.ndim == 2 and prod_minmax >= 0) or en1.ndim == 3 and all(prod_minmax >= 0):
+        return _x_bootstrap_pseudo_likelihood(en1, max_excitation, num_bootstrap_samples)
+
+    else:
+        # To do: Add to examples!
+        # Consider H(s) = - h s_1 - h s_3 + 2h s_2 - s_1 s_2 - s_2 s_3
+        # e.g. suppose the sampleset contains only + + and - - .
+        # Now we have the perturbative correction ..
+        # We can infer chi based on ground state manifold (only)! Assuming
+        # perturbative.
+        # There are no local excitations present in the sample set, therefore
+        # the temperature is estimated as 0.'
+        if en1.ndim == 2 or en1.shape[0] == 1:
+            # Scalar method 'inverse temperature' for one Hamiltonian
+            en1 = en1.reshape(en1.shape[-2:])  # f_{i}(s) - column i, row s.
+            if x0 is None:
+                x0 = -1 / max_excitation
+            elif hasattr(x0, "__iter__"):
+                x0 = x0[0]  # Scalar
+        else:
+            # Multi-parameter generalization
+            if x0 is None:
+                # Assume all bqms except the first are perturbative in absence
+                # of additional context.
+                x0 = np.zeros(en1.shape[0])
+                x0[0] = -1 / max_excitation[0]  # Smallest gap
+
+        d_mean_log_pseudo_likelihood = _create_d_mean_log_pseudo_likelihood(en1, degenerate_fields, sample_weights)
+
+        if optimize_method == "bisect" and en1.ndim == 2:
+            bisect_bracket = kwargs_opt.get("bisect_bracket", (1e-3, 1000))
```

Comment:
Switching out `bisect_bracket` for a general kwargs that can be past to SciPy's optimization methods.
```suggestion
            bisect_bracket = kwargs_opt.pop("bracket", (1e-3, 1000))
```